packages/framework/client-logger/app-insights-logger/package.json

```
@@ -39,6 +39,7 @@
 	},
 	"dependencies": {
 		"@fluidframework/core-interfaces": "workspace:~",
+		"@fluidframework/telemetry-utils": "workspace:2.0.0-internal.7.2.0",
```

After this, `pnpm i` again from the root to fix the lockfile, which also got the hardcoded version.
```suggestion
		"@fluidframework/telemetry-utils": "workspace:~",
```

---

Updated.